frontend/src/renderer/components/Process/Dropbox.svelte

```diff
@@ -45,13 +45,10 @@
     }
     .sectionStyle {
         background-color: #2c2c2f;
-        height: auto;
         width: 80%;
         border-radius: 10px;
         margin: auto;
-    }
-    .errorStyle {
-        background-color: red;
+        overflow: auto;
```

Comment:
Since errorStyle isn't getting applied (at least in my tests) I don't think this is doing what you expect it to

Reply:
I feel like you have not pulled the most recent changes as I do not have this code locally.